Source/ConversationV1/Conversation.swift

```diff
@@ -61,7 +61,11 @@ public class Conversation {
             let json = try JSON(data: data)
             let error = try json.getString(at: "error")
             let code = (try? json.getInt(at: "code")) ?? 400
-            let userInfo = [NSLocalizedFailureReasonErrorKey: error]
+            let description = try? json.getString(at: "description")
+            let userInfo = [
+                NSLocalizedFailureReasonErrorKey: error,
+                NSLocalizedRecoverySuggestionErrorKey: description
```

Comment:
Ah - this unfortunately escaped my notice until we started receiving build errors. Wasn't able to figure out where they were coming from until I downloaded Swift 3.1 development snapshot cause Xcode 8.2 is apparently buggy. 

In any case, you have` description` as an optional, and then `userInfo` as a non-optional. So when you return the `NSError` that includes the `userInfo`, the compiler complains this expression is that the `NSLocalizedRecoverySuggestionErrorKey` key of `UserInfo` is optional. Looking into how to resolve it now. 

I was thinking of adding a `guard let...else` statement to provide two different types of `userInfo` dictionaries, but let me know what you think, or if there's an alternative that's better.

At the same time, this might not even be why Travis is failing to build successfully since I ran these code changes with Travis prior to merging and the build was successful then. Shall find out :P